chat/public/css/chat.css

```diff
@@ -238,6 +242,14 @@ body.view-in-course .course-wrapper.chromeless {
     border-radius: 10px;
 }
 
+.chat-block .response-button.selected button {
+    opacity: 0.5;
+}
+
+.chat-block .response-button:not(.selected) button {
+    opacity: 1;
```

Comment:
If we change the `button:disabled` rule as mentioned above, we can remove this rule altogether.